saleor/graphql/checkout/tests/mutations/test_checkout_delivery_method_update.py

```diff
@@ -419,11 +419,8 @@ def test_checkout_delivery_method_update_external_shipping_invalid_currency(
     errors = data["errors"]
     assert len(errors) == 1
     assert errors[0]["field"] == "deliveryMethodId"
-    assert errors[0]["code"] == CheckoutErrorCode.DELIVERY_METHOD_NOT_APPLICABLE.name
-    assert (
-        errors[0]["message"]
-        == "Cannot choose shipping method with different currency than the checkout."
-    )
+    assert errors[0]["code"] == CheckoutErrorCode.NOT_FOUND.name
```

Comment:
Shouldn't it be described as breaking change? Someone could have implemented a notification based on this error code.

Reply:
Also, we should add it to the migration guide in [this PR. ](https://github.com/saleor/saleor-docs/pull/1473/files)

Reply:
It is not a breaking change from the API perspective. Anything that could cause such error is not possible anymore - we don't return external shipping with different currency than the checkout's currency. 

But adding it to migration guide is a valid point, in case someone would have an additional implementation, to filter out the shipping methods with incorrect currency on the frontend/app side. I've already added it :)